guides/setup-development.md

```diff
@@ -95,6 +93,30 @@ Then run the following command to populate the database:
 psql manage_courses_backend_development < ~/Downloads/backup_sanitised.sql
 ```
 
+## Seeding GiasSchool Data
+
+You can obtain an updated csv from https://www.get-information-schools.service.gov.uk/Downloads the `establishment fields CSV` checkbox. This can be uploaded to the `csv` directory.
```

Comment:
I wonder if we should split this text for running it with the current file (in which case you don't need to provide a csv path as it'll default) and then what to do if you need to use a different version?

Reply:
If we upload a new file, would we also want to update the default path to point at the new file, or leave the default pointing at the original?

Reply:
good point, we'd just update the path to point to the new file and swap it in. Then when running this in all other cases, you could just leave it to the defaults